lib/webapi/webapi.go

```diff
@@ -101,6 +101,9 @@ type Config struct {
 	// ServiceUser specifies the service user to use to
 	// create a cluster with for wizard-based installation
 	ServiceUser systeminfo.User
+	// InstallToken specifies the token to install cluster with.
+	// The token is used to authenticate agents during the install operation
+	InstallToken string
```

Comment:
webapi handler also handles installation and needs to pass the original token to the cluster so it could be used when the install operation is created. The token itself is used to fix the issue with wizard not properly authenticating agents.
I think I can do away with passing it into webapi by providing a wizard-specific Operator.